experimental/auth-logic/wrapper_interface.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package authlogic
```

Comment:
Could you add a short documentation for the package? :)

Reply:
fixed. Does this go in every file in the package?

Reply:
I found this useful: https://yaqs.corp.google.com/eng/q/5774342963855360#a5741031244955648

Based on this it is better to have an `authlogic.go` file containing the most important part of the functionality, and package documentation. Then the package documentation can be removed from the rest of the files. 
Currently, the `x_test.go` files all have package documentation too, which should be removed.